homeassistant/components/light/hyperion.py

```diff
@@ -28,7 +28,7 @@
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Required(CONF_HOST): cv.string,
     vol.Required(CONF_PORT, default=DEFAULT_PORT): cv.port,
-    vol.Optional(CONF_DEFAULT_COLOR, default=DEFAULT_COLOR): cv.string,
+    vol.Optional(CONF_DEFAULT_COLOR, default=DEFAULT_COLOR): cv.ensure_list,
```

Comment:
Hi @balloob,

I have tested this change and it does infact work, unfortunately i'm not that familiar with python to be able to just add validation, even with your example.

Sorry